packages/bloc/lib/src/bloc.ts

```diff
@@ -11,74 +13,35 @@ export type NextFunction<Event, State> = (value: Event) => Observable<Transition
  * @export
  * @abstract
  * @class Bloc
- * @extends {Observable<State>}
+ * @extends {BlocBase<State>}
  * @template Event
  * @template State
  */
-export abstract class Bloc<Event, State> extends Observable<State> {
-  constructor(private _state: State) {
-    super()
-    this.stateSubject = new Subject()
+export abstract class Bloc<Event, State> extends BlocBase<State> {
+  /* istanbul ignore next */
```

Comment:
```suggestion
```

Why are we ignoring this line?

Reply:
Just a problem with jest. 
Babel converts the super to `var _this = _super.call (this) || this` and this will result in never testing the second condition.